application/views/userManagement/index.php

```diff
@@ -31,6 +31,8 @@
             'lsAfterAjaxUpdate' => [
                 'bindListItemclick();',
                 'LS.UserManagement.bindButtons();',
+                'showDeactivatedUserTooltip();',
+                'LS.UserManagement.updateUserLimit();',
```

Comment:
Not part of this PR

Reply:
Need an js event where plugin can add own js action ?

something like $("#grid").trigger("gridUpdated:end") ([see why adding :](https://learn.jquery.com/events/introduction-to-custom-events/#naming-custom-events))